magclassic/templates/emails/checkin_barcode.html

```diff
@@ -0,0 +1,15 @@
+<html>
+<head></head>
+<body>
+
+{{ attendee.first_name }},
+
```

Comment:
I recommend mentioning that this is optional, so that people who forget it think they're in trouble or anything like that.  Instead we can mention that you'll get your badge faster if you bring this.